internal/k8shandler/elasticsearch_test.go

```diff
@@ -0,0 +1,111 @@
+package k8shandler
+
+import (
+	"net/http"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/elasticsearch-operator/internal/constants"
+	"github.com/openshift/elasticsearch-operator/internal/elasticsearch"
+	"github.com/openshift/elasticsearch-operator/test/helpers"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+var _ = Describe("Elasticsearch", func() {
+	defer GinkgoRecover()
```

Comment:
the tests look great... but is it possible to rewrite them in not ginko?

Reply:
Changed.